dev/releases/PackageInfos-to-JSON.g

```diff
@@ -0,0 +1,11 @@
+LoadPackage("json");
+
+InstallMethod(_GapToJsonStreamInternal,
+[IsOutputStream, IsObject],
+function(o, x)
+    PrintTo(o, "null");
+end);
+
+out := OutputTextFile("package-infos.json", false);
+GapToJsonStream(out, GAPInfo.PackagesInfo);;
+QUIT;
```

Comment:
How about printing to stdout instead? Then the code calling it has full control over where the output goes; and only one script needs to contain the name of the JSON file